contracts/src/v0.8/functions/dev/1_0_0/FunctionsSubscriptions.sol

```diff
@@ -526,14 +528,15 @@ abstract contract FunctionsSubscriptions is IFunctionsSubscriptions, ERC677Recei
       }
 
       IFunctionsBilling coordinator = IFunctionsBilling(request.coordinator);
+      coordinator.deleteCommitment(requestId);
```

Comment:
I like your thinking, I should clarify this change:
So I removed the check for the return value because:
The Router is the source of truth. It shouldn't be dependent on the Coordinator doing an action for it to continue. In the future where Coordinator contract might be DON owned, what if the Coordinator always returned false here? Then users could never get their blocked funds back.

Reply:
Who owns coordinator contracts? Are you implying the user cannot get funds back if the DON owned coordinator is malicious? That sounds out of scope

Reply:
This made me thing of another case where that we should make sure we think through carefully:
1. The DON attempts to call the user's callback, but fails for some reason that is not the fault of the user (ie: user doesn't get charged)
2. Since we delete the commitment, how can a user time out their request and have their funds unblocked?

Reply:
In the above the user's funds will unblocked, because the Router handles subscriptions

Reply:
@KuphJr which row of the fulfillment events doc is that scenario in? We can discuss over there too.

Reply:
> Who owns coordinator contracts? Are you implying the user cannot get funds back if the DON owned coordinator is malicious? That sounds out of scope
Currently CLL, but yeah the hope is eventually DON owned.
The overall design was already heavily influenced by building toward this :shrug: